Tests/TestingApps/PaywallsTester/PaywallsTester.xcodeproj/xcshareddata/xcschemes/PaywallsTester - SK config.xcscheme

```diff
@@ -86,7 +86,7 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <StoreKitConfigurationFileReference
-         identifier = "../PaywallsTester/Products.storekit">
+         identifier = "../Tests/TestingApps/PaywallsTester/PaywallsTester/Products.storekit">
```

Comment:
Cleans up the relative directory so it picks up correctly when opening the workspace, which is the only thing we should be using from now on